src/app/app.component.scss

```diff
@@ -1,18 +1 @@
 @import '~@pxblue/colors/palette.scss';
```

Comment:
probably don't need this in here if it's not used for any styles...file eliminated!